.github/pull_request_template.md

```diff
@@ -0,0 +1,5 @@
+# What
+
+# Why
+
+:warning: This application is now continuously deployed. Merged changes are automatically deployed to staging and production. Only merge changes if you are happy for them to go live. :warning:
```

Comment:
We should refer to the docs here, to provide more context and pointers.

Reply:
Sure can we just add this in for now and update later?

Reply:
Hmm...I don't think this is that urgent. I was just asking to add a link to here: https://docs.publishing.service.gov.uk/manual/development-pipeline.html#merge-your-own-pull-request